static/js/popovers.js

```diff
@@ -926,8 +1054,28 @@ exports.register_click_handlers = function () {
     });
 
     $('body').on('click', '.flatpickr-confirm', function (e) {
-        var datestr = $(".remind.custom")[0].value;
-        reminder_click_handler(datestr, e);
+        var datestr;
+        if (send_later_popover_elem !== undefined) {
+            var instance = $(current_flatpickr_instance);
+            var date = instance.prop("selectedDates");
+            datestr = flatpickr.formatDate(date[0], "M d, Y at h:i ") + flatpickr.formatDate(date[0], "K");
+            var message_content = compose_state.message_content();
+
+            var command = "/schedule " + datestr + "\n";
+
+            if (reminder.is_deferred_delivery(message_content)) {
+                var raw_message = message_content.split('\n');
+                message_content = raw_message.slice(1).join('\n');
+            }
+            message_content = command + message_content;
+            compose_state.message_content(message_content);
+            popovers.hide_send_later_popover();
```

Comment:
Same story with this as with the below; we should just call a common function rather than duplicating code.